### ISSUE TITLE
Discuss including clip arithmetic operator

### ISSUE THREAD

**Issue by amoeba** [substrait-io/substrait#366]
[clip](https://numpy.org/doc/stable/reference/generated/numpy.clip.html) is an uncommon but handy arithmetic operation that can as be expressed as a combination of the more common operations [least/greatest](https://www.postgresql.org/docs/current/functions-conditional.html#FUNCTIONS-GREATEST-LEAST). None of these exist in Substrait though there's an open PR for [least/greatest](https://github.com/substrait-io/substrait/pull/247).

For example,

```python
import numpy as np

a = np.arange(10) # => array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])

np.minimum(np.maximum(a, 4), 6) # => array([4, 4, 4, 4, 4, 5, 6, 6, 6, 6])
np.clip(a, 4, 6) # => array([4, 4, 4, 4, 4, 5, 6, 6, 6, 6])
```

https://github.com/substrait-io/substrait/issues/307#issue-1360543636 sets out to establish guidelines on scope which leads me to think this might be too specialized to be included in Substrait but I thought I'd post an issue to get some feedback on whether this might be on scope for Substrait.

**Comment by westonpace** [comment]
I agree this is too specialized given the guidance in #307 .  However, it seems generally useful.  I'm going to close this issue.  If we want to pursue this function then I would encourage you to create a new project with a mandate defined similar to #307.  This can then be an issue on that new project.

**Comment by amoeba** [comment]
Thanks for circling back on this, @westonpace. I don't have any plans to pursue this at the moment.
